src/app/(auth)/components/SocialButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+interface SocialButtonProps {
+  color: string;
+  iconPath: string;
+  socialName: string;
+  onClick: () => void;
+}
+
+export default function SocialButton({ color, iconPath, socialName, onClick }: SocialButtonProps) {
+  return (
+    <Button
+      variant="contained"
+      style={{ borderRadius: '24px', backgroundColor: color,}}
+      className='font-medium font-roboto h-12 w-36'
```

Comment:
I think you can remove this and use a single font across all the components and pages